Add Engineering Manager JD (#2)
orderlymeds/hiring: roles/engineering_manager.md

**What we offer:**

- Flexible remote work environment
- Competitive salary and benefits package
- Equity in a fast-growing company with no VC funding dilution
- Professional development opportunities, including training and certifications
- A collaborative and inclusive work culture
- Opportunities to work on challenging and meaningful projects

**What we value:**

- Customer focus and maintaining empathy with our customers.
- Asynchronous communication by default with synchronous collaboration when needed.
- Over-communication and collaborative debugging
- Autonomy and action over consensus
- Ownership over the product and the work
- Collaboration and supporting team members
- Making work visible
- Iterating continuously

**How we work:**

- Ideas and priorities flow from across the company.
- Big initiatives are planned into 4-8 week releases.
- Progress is tracked in two-week cycles.
- Merged PRs auto-deploy to staging and production via chatops.
- Recorded work in progress demos are shared with the company.

**Key responsibilities:**

> - Lead the main engineering team to develop scalable, reliable, and maintainable systems

Reviewer: ```suggestion
- Empower the engineering team to develop scalable, reliable, and maintainable systems
```

> - Promote good engineering practices within the team and beyond

Reviewer: ```suggestion
- Contribute via leveraged output activities -- one-on-ones, training, pair programming, and adding CI checks.
- Promote good engineering practices within the team and beyond
```

- Identify and implement improvements to our products and processes to support our growth

> - Collaborate with cross-functional teams to ensure technical solutions align with product strategy

Reviewer: ```suggestion
- Collaborate with cross-functional teams to ensure technical solutions align with product strategy and reasonable timelines
```

> - Ensure team members understand how the work they are doing ties to wider company goals

Author: Or maybe "Roadmap and backlog management; communicating priorities to team and keeping on track"

- Manage a remote team across multiple timezones
- Nurture a healthy team culture by

>   - role modelling the behaviours you wish to see,

Reviewer: ```suggestion
  - being a role model for the behaviors you wish to see,
```

  - coach team members around work and personal growth,
  - seek to create and support a diverse and inclusive team, and
  - monitor team health and morale and ensure improvements
- Foster a continuous learning and improvement mindset within your team. Ensure team members have the skills needed for their role
- Be a sponsor for engineers so they have access to growth opportunities they wouldn’t otherwise have on their own

**Your peers would describe you as:**

> - Dependable in meeting your commitments

Reviewer: ```suggestion
- Extremely dependable in meeting your commitments
```

Author: Only one I'm going to keep as is. I don't think the Extremely is needed
